### PR TITLE
Added support for CronJobs

### PR DESCRIPTION
**Pull request by somayaj**

### CODE REVIEW

File: src/app/cronjobs.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+use k8s_openapi::{api::batch::v2alpha1::CronJob, chrono::Utc};
+
+use super::{models::KubeResource, utils};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct KubeCronJob {
+  pub name: String,
```
Comment (deepu105) [inline src/app/cronjobs.rs:7]: need to align with what kubectl shows

File: test_data/cronjobs.yaml [hunk 2]
```diff
@@ -0,0 +1,331 @@
+apiVersion: v1
```
Comment (deepu105) [inline test_data/cronjobs.yaml:1]: should be generated form `kubectl get cronjobs -o yaml`

**Review by deepu105 — CHANGES_REQUESTED** [review]
some comments, also will wait for the UI part to be added

**Comment by deepu105** [comment]
@somayaj I fixed the parsing and tests. Can you add the section to the UI please

**Comment by somayaj** [comment]
i didn;t get that actually. What files are those?

**Comment by deepu105** [comment]
I mean to make cronjobs appear in the UI, you need to add the rendering code under ui/overview.rs. you can follow what is already there for jobs and replicate that

**Comment by somayaj** [comment]
> I mean to make cronjobs appear in the UI, you need to add the rendering code under ui/overview.rs. you can follow what is already there for jobs and replicate that

My bad, I thought I had those in the initial commit. I've checked them in now.

File: src/ui/overview.rs [hunk 3]
```diff
@@ -844,6 +870,52 @@ fn draw_jobs_block<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect) {
...
+        "Namespace",
+        "Name",
+        "Schedule",
+        "Last Scheduled",
+        "Suspend",
+        "Active",
+        "Age",
+      ],
+      column_widths: vec![
+        Constraint::Percentage(25),
```
Comment (deepu105) [inline src/ui/overview.rs:893]: Total percentage needs to be 99 else we will have out of bound errors

**Review by deepu105 — CHANGES_REQUESTED** [review]

**Review by deepu105 — APPROVED** [review]

**Pull request merged**